### PR TITLE
SAAS-774 Adds email to contacts page

### PR DESCRIPTION
**Pull request by tiagomotasantos**
Adds email to contacts page

https://jira.percona.com/browse/SAAS-774

Screenshot 2021-12-07 at 11 59 49

Screenshot 2021-12-07 at 12 23 26

### CODE REVIEW

File: src/pages/Dashboard/Contacts/Contacts.tsx [hunk 1]
```diff
@@ -110,6 +110,11 @@ export const Contacts: FC = () => {
       <div className={styles.card}>
       <Overlay className={styles.cardOverlay} isPending={loading}>
         <p className={styles.cardTitle}>{Messages.perconaContacts}</p>
+        <p>
+          {Messages.needHelp}
+          &nbsp;
```
Comment (nicolalamacchia) [inline src/pages/Dashboard/Contacts/Contacts.tsx:115]: nit: please replace the non-breaking space with a margin in styles

File: src/pages/Dashboard/Contacts/Contacts.tsx [hunk 2]
```diff
@@ -110,6 +110,11 @@ export const Contacts: FC = () => {
...
+          <a className={styles.externalLink} href={`mailto: ${HELP_EMAIL}`}>{HELP_EMAIL}</a>
```
Comment (nicolalamacchia) [inline src/pages/Dashboard/Contacts/Contacts.tsx:116]: Please remove the space after `mailto:`

**Review by nicolalamacchia — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard/Contacts/Contacts.tsx (hunk 2 above)
Comment (tiagomotasantos) [reply src/pages/Dashboard/Contacts/Contacts.tsx]: Done

File: src/pages/Dashboard/Contacts/Contacts.tsx (hunk 1 above)
Comment (tiagomotasantos) [reply src/pages/Dashboard/Contacts/Contacts.tsx]: Done

**Pull request merged**